pkg/apiserver/config.go

```diff
@@ -16,6 +16,44 @@ limitations under the License.
 
 package apiserver
 
+import (
+	"k8s.io/apimachinery/pkg/apimachinery/announced"
+	"k8s.io/apimachinery/pkg/apimachinery/registered"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
+)
+
+var (
+	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
+	registry             = registered.NewOrDie("")
+	// Scheme for API object types
+	Scheme = runtime.NewScheme()
+	// Codecs for creating a server config
+	Codecs = serializer.NewCodecFactory(Scheme)
+)
+
+func init() {
```

Comment:
Actually this method invokes apimachinery `AddToGroupVersion` (https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/register.go#L43) which should register the `ExportOptions` type, so I'm not sure why I see the error in the log mentioned above.